addons/binding/org.openhab.binding.hue/README.md

```diff
@@ -224,14 +225,13 @@ And there is one Hue Motion Sensor (represented by three devices) and a Hue Dimm
 ### demo.things:
 
 ```
-Bridge hue:bridge:1 [ ipAddress="192.168.0.64" ] {
-    0210 bulb1 [ lightId="1" ]
-    0220 bulb2 [ lightId="2" fadetime=300 ]
-    0106 light-level-sensor [ sensorId="3" ]
-    0107 motion-sensor [ sensorId="4" ]
-    0302 temperature-sensor [ sensorId="5" ]
-    0820 dimmer-switch [ sensorId="6" ]
-}
```

Comment:
It does matter that the `}` is now missing. :wink:

Reply:
Oh, yes, I missed that. Thanks.

Reply:
fixed!